Simulator/Vertices/Neuro/AllSpikingNeurons.cpp

```diff
@@ -17,8 +18,12 @@ void AllSpikingNeurons::setupVertices()
    int maxSpikes = static_cast<int>(Simulator::getInstance().getEpochDuration()
                                     * Simulator::getInstance().getMaxFiringRate());
 
-   hasFired_.assign(size_, false);
-   vertexEvents_.assign(size_, maxSpikes);
+   hasFired_.resize(size_, false);
+   vertexEvents_.resize(size_, maxSpikes);
```

Comment:
It's possible that @KamathScience  decided on `assign()`, rather than `resize()`; I'm not sure — consult with her.